docs/conch/examples/sshsimpleclient.py

```diff
@@ -143,6 +156,20 @@ def closed(self):
         reactor.stop()
 
 
-log.startLogging(sys.stdout)
-protocol.ClientCreator(reactor, SimpleTransport).connectTCP(HOST, PORT)
-reactor.run()
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Simple SSH client.")
+
+    parser.add_argument(
+        "--client_public_key_path",
```

Comment:
I think usually a dash is used for arguments. You can then use the `dest` keyword to convert it to an underscore, though I am not sure whether `argparse` would do that automatically.